pymc3/variational/svgd.py

```diff
@@ -7,11 +7,13 @@
 import theano
 import theano.tensor as tt
 from tqdm import tqdm
+from updates import *
```

Comment:
I would guess this would have to change to either `from .updates` or `from pymc3.variational.updates`